.gitignore

```diff
@@ -41,3 +41,4 @@ nb-configuration.xml
 .sdkmanrc
 .envrc
 .jekyll-cache
+vale.ini
```

Comment:
```suggestion
.vale.ini
```
Do you want to include the dot prefix in the file name?

Reply:
@rolfedh - oops. Didn't mean to commit this config change. I've reversed this bit of the commit and added the config to my local global git ignore file. Thanks for flagging though :+1: 

In the Quarkus implementation of Vale, the config file is without the preceding dot (.). I use it in the root of my local IDE for testing the rule tweaks.

![image](https://user-images.githubusercontent.com/92924207/211526827-b258a596-022a-4553-9b1a-c63500f954e0.png)